src/helix-ui/styles/core/hx-checkbox.less

```diff
@@ -0,0 +1,82 @@
+hx-checkbox {
```

Comment:
The majority of the styles for `hx-checkbox` were applied to `:host` in the shadow dom (which is overridable), so they've been pulled out into core CSS to be applied in the light dom (which is also overridable).